### ISSUE TITLE
Twitter API

### ISSUE THREAD

**Issue by anthonymasure** [l-atelier-des-chercheurs/Les-Cahiers-du-Studio#51]
Utiliser l'API Twitter pour récupérer automatiquement les tweets d'un hashtag et les placer sur la timeline. Possibilité de supprimer des tweets si non pertinents ou spam. Prévoir l'affichage des médias joints dans les tweets (photos, etc.).

**Comment by louis-ev** [comment]
Merci pour les retours !

On peut commencer par ajouter un bouton "tweet" qui accepte une URL et importe un tweet dans la timeline, non ? L’idéal serait de copier son contenu dans sa globalité pour que ça fonctionne aussi hors ligne, non ?

**Comment by anthonymasure** [comment]
Ca me paraît un peu fastidieux d'importer les tweets un par un, alors que la puissance du LT c'est de tout agréger. Par contre il faut qu'on puisse trier / supprimer. Mais dans un 1er temps si on peut importer un par un ça serait déjà très cool ! Oui, il faut que ça puisse fonctionner hors ligne...

**Comment by louis-ev** [comment]
Je vois bien l’intérêt d’importer un LT. Mais ça oblige vraiment à envisager un système de sélection avant import, il y a trop de risques d’importer des messages non-pertinents.

À voir comment s’y prendre du coup, ça demander d’ajouter un système de login avec autorisation — pas évident.

**Comment by anthonymasure** [comment]
Oui, c'est plus lourd en terme d'interface et de process. Je pense que les deux peuvent approches, à terme, peuvent cohabiter (import tweet par tweet, et tableau de bord)
